sdk/golang/cima/sdk.go

```diff
@@ -19,14 +20,17 @@ type SDK struct {
 }
 
 // GetCCReport implements EvidenceAPI
-func (s *SDK) GetCCReport(nonce, userData string, extraArgs map[string]any) (evidence_api.Report, error) {
+func (s *SDK) GetCCReport(nonce, userData []byte, extraArgs map[string]any) (evidence_api.Report, error) {
 	client, err := NewClient()
 	if err != nil {
 		log.Fatalf("[GetCCReport] failed to connect to client with error %v", err)
 		return nil, err
 	}
 
-	result, err := client.GetCCReportFromServer(userData, nonce)
+	nonceStr := b64.StdEncoding.EncodeToString(nonce)
+	userDataStr := b64.StdEncoding.EncodeToString(userData)
+
+	result, err := client.GetCCReportFromServer(userDataStr, nonceStr)
```

Comment:
Can we modify client.GetCCReportFromServer API also.
I can direct use the GetCCReportFromServer in the ITA client.

Reply:
In side the GetCCReportFromServer func, can we make the container id to be optional? 
If we provide the container id, can we add it at the end of the userData?

Reply:
> Can we modify client.GetCCReportFromServer API also. I can directly use the GetCCReportFromServer in the ITA client.

The GetCCReportFromServer is currently not an API exposed in CIMA I think. Are you trying to fetch vm level measurement? If that's the case, i think using the vmsdk is a better choice. @dongx1x Please add more comments here.

Reply:
@rihuita maybe you can use `GetCCReport` instead of `GetCCReportFromServer`, since GetCCReportFromServer is designed for transparent communication between client and server. The `container id` can NOT be optional for container measurement, but there is a parameter to customize the measurement: https://github.com/cc-api/container-integrity-measurement-agent/blob/main/service/cima-server/configs/policy.yaml#L11